panel/widgets/slider.py

```diff
@@ -319,6 +320,36 @@ def _process_property_change(self, msg):
         return msg
 
 
+class DatetimeSlider(DateSlider):
+    """
+    The DatetimeSlider widget allows selecting a value within a set of
+    bounds using a slider.  Supports datetime.datetime
```

Comment:
```suggestion
    bounds using a slider. Supports datetime.datetime
```

Very nit.

Reply:
@hoxbro no, that's a good one. I also re-added `datetime.date` to list of supported dtypes to the docs.